src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,197 @@
+                Files.createFile(pathToFile);
+                System.out.printf("%sFile tasks.csv successfully created%s\n", GREEN_BOLD_BRIGHT, RESET);
+            }
+            return tasks;
+        } catch (IOException e) {
+            System.err.println("Error while reading tasks.csv file");
+            e.printStackTrace(System.err);
+            System.err.println("Program will try create new file");
+            try {
+                Files.createFile(pathToFile);
+            } catch (IOException ex) {
+                throw new RuntimeException(ex);
+            }
+            System.out.printf("%sFile tasks.csv successfully created%s\n", GREEN_BOLD_BRIGHT, RESET);
+        }
+        return tasks;
+    }
+
+    private static void displayMenu() {
+        System.out.printf("\n%s#  Welcome to the Task Manager  #\n", GREEN_BOLD_BRIGHT);
+        System.out.print("#                               #\n");
+        System.out.printf("#   %s1. Add a new task:  %sadd     %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s2. Remove a task:   %sremove  %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s3. Show all tasks:  %slist    %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#   %s4. Save and Exit:   %sexit    %s#\n", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, GREEN_BOLD_BRIGHT);
+        System.out.printf("#################################%s\n", RESET);
+    }
+
+    private static void showTasks() {
+        System.out.printf("\n%sTask List:%s\n", GREEN_BOLD_BRIGHT, RESET);
+        String[] task;
+        for (int i = 0; i < tasks.length; i++) {//this loop controlling if task is important or not for different colors in output
+            task = tasks[i];
+            switch (task[2].trim()) {
+                case "true" ->
+                        System.out.printf("%s%d. %s%s%s\n", GREEN_BOLD_BRIGHT, i + 1, RED_BOLD_BRIGHT, String.join(",", task), RESET);
+                case "false" ->
+                        System.out.printf("%s%d. %s%s%s\n", GREEN_BOLD_BRIGHT, i + 1, BLUE_BOLD_BRIGHT, String.join(",", task), RESET);
+            }
+        }
+    }
+
+    private static void addTask() {
+        final var sizeTaskData = 3;
+        final int taskNum = tasks.length + 1;
+        final String[] task = new String[sizeTaskData];
+
+        System.out.printf("\n%sAdding new task Number.%d:%s\n", GREEN_BOLD_BRIGHT, taskNum, RESET);
+        System.out.printf("%sWrite the description of the task:%s ", BLUE_BOLD_BRIGHT, RESET);
+        task[0] = scan.nextLine();
+
+        System.out.printf("%sWrite the date when task end -in the format %sYYYY-MM-DD%s:%s ", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+        task[1] = " %s".formatted(dateChecker());//because space after comma
```

Comment:
CLEAN-CODE: lepsi pojmenovani loadAndCheckDate()
```suggestion
        task[1] = " %s".formatted(loadAndCheckDate());//because space after comma
```